jujugui/static/gui/src/app/components/added-services-list/added-services-list.js

```diff
@@ -32,7 +32,7 @@ YUI.add('added-services-list', function() {
               // of the final deployed service id structure and we want react
               // to treat them as the same record instead of re-rendering
               // when they key changes.
-              key={service.get('name')}
+              key={service.get('id')}
```

Comment:
Is this comment no longer valid?
 ```
               // We use the 'name' instead of the 'id' here because when a
               // ghost service is added it uses the ghost id structure instead
               // of the final deployed service id structure and we want react
               // to treat them as the same record instead of re-rendering
               // when they key changes.
```